index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="style.css">
+
+    <title>Jonathan Sivahera</title>
+  </head>
+  <body>
+    <div class="blur-item circle">
+      
+    </div>
+    <!-- Header section -->
+    <header class="header-section">
+    <a href="index.html#home">JS</a>
+    <div class="hamburger"></div>
+    </header>
+    <!-- Main page content -->
+    <main class="main-section" id="home">
+      <h1 class="titles" id="main-title">Jonathan Sivahera</h1>
+      <h2 class="titles" id="main-sub">Software Developer</h2>
+      <p class="intro">
+        I can help you build a product,
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded,  
+        don’t hestiate to contact me.
+      </p>
```

Comment:
Fantastic job! Well done implementing this section's elements with the proper tags 😎